zproject/default_settings.py

```diff
@@ -218,9 +218,22 @@
 AVATAR_CHANGES_DISABLED = False
 PASSWORD_MIN_LENGTH = 6
 PASSWORD_MIN_GUESSES = 10000
-PUSH_NOTIFICATION_BOUNCER_URL: str | None = None
+
+ZULIP_SERVICES_URL = "https://push.zulipchat.com"
+ZULIP_SERVICE_PUSH_NOTIFICATIONS = False
+ZULIP_SERVICE_SUBMIT_USAGE_STATISTICS = False
+ZULIP_SERVICE_SECURITY_ALERTS = False
```

Comment:
So it felt a bit controversial whether these should be default `False` or `None`:
- Having `None` would allow us to actually know that the server doesn't set these settings at all (vs setting them, but to `False`)
- It makes the type `bool | None`, which is annoying and I don't feel this actually brings a benefit.

Currently I made these `False` and if all of them are `False`, `computed_settings` will treat this as if "not set"  and do the backward compatibility code to see if  the `PUSH_BOUNCER_URL` and `SUBMIT_USAGE_STATISTICS` are activated. This is probably fine behavior?